opm/simulators/linalg/amgcpr.hh

```diff
@@ -841,6 +851,7 @@ namespace Dune
 
     template<class M, class X, class S, class PI, class A>
     void AMGCPR<M,X,S,PI,A>::mgc(LevelContext& levelContext){
+      //OPM_TIMEBLOCK(mgc);
```

Comment:
Is this commented out intentionally

Reply:
yes. Some of the macros have troubles if functions are to simple. Scope probably is removed ??